src/databricks/sqlalchemy/requirements.py

```diff
@@ -1,20 +1,4 @@
 """
-This module is supposedly used by the compliance tests to control which tests are run based on database capabilities.
-However, based on some experimentation that does not appear to be consistently the case. Until we better understand
-when these requirements are and are not implemented, we prefer to manually capture the exact nature of the failures
-and errors.
-
-Once we better understand how to use requirements.py, an example exclusion will look like this:
-
-    import sqlalchemy.testing.requirements
-    import sqlalchemy.testing.exclusions
-
-    class Requirements(sqlalchemy.testing.requirements.SuiteRequirements):
-        @property
-        def __some_example_requirement(self):
-            return sqlalchemy.testing.exclusions.closed
-
-
```

Comment:
I removed this comment since we now have a better understanding of how to use `requirements.py` to dictate which features are skipped.

That said, I think our approach going forward should be to use `requirements.py` only to _permit_ tests to run (i.e. only to include exclusions.open() calls). For the tests that we skip because of limitations of Databricks, I think we should manually call these out in `test_suite.py` and provide a more precise skip message. Because if the test is skipped because of `requirements.py`, the output from pytest simply says "test not enabled for any dialect" which could be misleading.

Reply:
I've implemented this fix in b31eef45bb56e9d02253e0924896e999b8c5ab38 and 8200e23159d3b65e9e92320133135a39fabce93d 👍